impeller/playground/switches.cc

```diff
@@ -13,6 +13,7 @@ PlaygroundSwitches::PlaygroundSwitches(const fml::CommandLine& args) {
   if (args.GetOptionValue("playground_timeout_ms", &timeout_str)) {
     timeout = std::chrono::milliseconds(atoi(timeout_str.c_str()));
   }
+  enable_vulkan_validation = args.HasOption("enable_vulkan_validation");
```

Comment:
Nit: It's possible to test this flag flip in a reasonable way via `switches_unittests.cc` these days